src/Xamarin.Android.Build.Tasks/Tasks/Aot.cs

```diff
@@ -282,12 +275,15 @@ bool RunParallelAotCompiler (List<string> nativeLibs)
 							return;
 						}
 
-						if (!RunAotCompiler (config.AssembliesPath, config.AotCompiler, config.AotOptions, config.AssemblyPath)) {
+						if (!RunAotCompiler (config.AssembliesPath, config.AotCompiler, config.AotOptions, config.AssemblyPath, config.ResponseFile)) {
 							LogCodedError ("XA3001", "Could not AOT the assembly: {0}", Path.GetFileName (config.AssemblyPath));
 							Cancel ();
 							return;
 						}
 
+						if (File.Exists (config.ResponseFile))
+							File.Delete (config.ResponseFile);
```

Comment:
You might not actually need this `Exists` check, since it won't throw if the file doesn't exist: https://docs.microsoft.com/en-us/dotnet/api/system.io.file.delete?view=netframework-4.8#remarks

Reply:
I always forget that...